src/index.html

```diff
@@ -11,12 +11,103 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#home"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="navbar">
+        <ul class="navbar-nav">
+          <li class="nav-item">
+            <a
+              href="#apple"
+              class="nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav-item">
+            <a
+              href="#samsung"
+              class="nav-link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav-item">
+            <a
+              href="#smartphones"
+              class="nav-link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav-item">
+            <a
+              href="##laptops-computers"
```

Comment:
Don't use spaces in `<a>` tag's `href` property. In line 70, there is a double hash (`##`) in the `href` attribute. It should be a single hash (`#`) followed by the anchor link.